editor/src/core/es-modules/package-manager/package-manager.spec.ts

```diff
@@ -196,7 +202,7 @@ describe('ES Dependency Manager — Real-life packages', () => {
     expect(onRemoteModuleDownload).toBeCalledTimes(0)
   })
 
-  it('antd@4.2.5', async (done) => {
+  it('antd@4.2.5', (done) => {
```

Comment:
in Jest 27, you can't write a test that is `async (done) =>`  you need to pick async or done()